docs/antrea-network-policy.md

```diff
@@ -696,30 +741,39 @@ to select Pods. More details can be found in the [ServiceAccountSelector](#servi
 **Note**: The order in which the egress rules are specified matters, i.e., rules will
 be enforced in the order in which they are written.
 
-**enableLogging**: Antrea-native policy ingress or egress rules can be
-audited by enabling its logging field. When `enableLogging` field is set to
-true, the first packet of any connection that matches this rule will be logged
-to a separate file (`/var/log/antrea/networkpolicy/np.log`) on the Node on
-which the rule is applied. These log files can then be retrieved for further
-analysis. By default, rules are not logged. The example policy logs all
-traffic that matches the "DropToThirdParty" egress rule, while the rule
-"AllowFromFrontend" is not logged. Specifically for drop and reject rules,
-deduplication is applied to reduce duplicated logs, and duplication buffer
-length is set to 1 second. If a rule name is not provided, an identifiable
-name will be generated for the rule and displayed in the log. For rules in layer
-7 NetworkPolicy, packets are logged with action `Redirect` prior to analysis by
-the layer 7 engine, more details are available in the corresponding engine logs.
-The rules are logged in the following format:
+**logSetting** and **enableLogging**: Antrea-native policy ingress or egress rules
+can be audited by enabling its logging field. Since v0.11.0, `enableLogging`
+became available and `logSetting` is introduced in v1.12.0. We will be deprecating
+`enableLogging` for `logSetting` so they should not be set at the same time.
+`logSetting.enabled` and `enableLogging` are equivalent. When `logSetting.enabled`
+or `enableLogging` field is set to true, the first packet of any connection that
+matches this rule will be logged to a separate file (`/var/log/antrea/networkpolicy/np.log`)
+on the Node on which the rule is applied. These log files can then be retrieved for
+further analysis. By default, rules are not logged. When using `logSetting`,
```

Comment:
Probably remove "By default, rules are not logged.".

Or move it to be before "When `logSetting.enabled` or `enableLogging` field is set to `true`", but then probably you should remove "`logSetting.enabled` and `enableLogging` are equivalent".